apps/chat/src/components/Common/Forms/FieldWarningMessage.tsx

```diff
@@ -0,0 +1,44 @@
+import { ComponentType, forwardRef } from 'react';
+
+import classNames from 'classnames';
+
+import { useTranslation } from '@/src/hooks/useTranslation';
+
+import { Translation } from '@/src/types/translation';
+
+interface Props {
+  warning?: string;
+  className?: string;
+}
+
+export const FieldWarningMessage = ({ warning, className }: Props) => {
+  const { t } = useTranslation(Translation.Settings);
+
+  if (!warning) {
+    return null;
+  }
+
+  return (
+    <div className={classNames('text-xxs text-warning', className)}>
+      {t(warning)}
+    </div>
+  );
+};
+
+export function withWarningMessage<T extends object, R>(
+  Component: ComponentType<T>,
+) {
+  const WarningMessageWrapper = forwardRef<R, Omit<Props, 'className'> & T>(
+    (props, ref) => (
+      <div>
```

Comment:
maybe better wrap by Fragment
```suggestion
      <>
```